CHANGELOG.md

```diff
@@ -484,7 +484,7 @@ v3.0.3 (pulled from pypi due to a packaging issue) ======
 
 v1.15.7 ======
 
--   Fix #174 with #207: Re-use samefile backport as developed in
+-   Fix #174 with #207: Reuse samefile backport as developed in
```

Comment:
While re-use is acceptable:
* Majors dictionaries suggest `reuse` only:
  * [OED](https://www.oed.com/search/dictionary/?q=re-use)
  * [Cambridge](https://www.oxfordlearnersdictionaries.com/definition/english/reuse_1)
  * [Collins](https://www.collinsdictionary.com/dictionary/english/reuse)
  * [Longman](https://www.ldoceonline.com/dictionary/reuse)
  * [Merriam-Webster](https://www.merriam-webster.com/dictionary/reuse)
  * [Oxford Learner's Dictionary](https://dictionary.cambridge.org/dictionary/english/reuse)
* From [The Blue Book of Grammar and Punctuation](https://www.grammarbook.com/blog/hyphens/hyphens-with-the-prefix-re/):
  > Use the hyphen with the prefix _re_ only when _re_ means _again_ AND omitting the hyphen would cause confusion with another word.